src/app/pages/component-viewer/component-viewer.scss

```diff
@@ -2,25 +2,20 @@
 
 app-component-viewer {
   font-weight: 400;
-  padding: 20px $content-padding-side 50px;
+  padding: 20px 50px 50px 20px;
```

Comment:
Maybe add a comment? Better yet, I see 20px, 25px, 50px coming up a lot. Could they be added to `_contants.scss`?

Reply:
I did a search and didn't find any other cases, can you point me to the other usages?

Reply:
I may have said that prematurely. I'll take a look when I get a chance to run it locally. Or I'll try to add sass constants throughout the app in a later PR.